Gemfile

```diff
@@ -35,7 +35,7 @@ gem "rdoc"
 gem "rest-client", require: "rest_client"
 gem "roadie-rails"
 gem "shoryuken", "~> 4.0", require: false
-gem "statsd-instrument", "~> 2.3.0"
+gem "statsd-instrument"
```

Comment:
can you pessimize this so we don't end up in nightmare resolver land where it tries every version that has ever existed any time there's a conflict? Looks like `~> 3.5` would give you the version you want and also give Bundler the smallest number of versions to try in a failure.

Reply:
in general we don't really seem to have requirements added to the dependencies in the gemfile

Reply:
You are correct, and that is how you get Gemfiles that take a billion years to resolve as soon as there's finally a conflict--I want to move in the direction of having more pessimized constraints over time, not less. If I have time, I'll run `pessimize` myself and check in the results, but in the meantime, let's not make the problem space bigger.